lib/containers/container_images.pm

```diff
@@ -57,10 +59,12 @@ sub build_with_sle2docker {
 # Testing openSUSE based images
 sub test_opensuse_based_image {
     my $image = $_[0];
+    die "You must define the image!" unless $image;
     my ($name, $tag) = split(/:/, $image);
     $tag //= 'latest';
 
     my $runtime = $_[1] //= "docker";
+    die "You must define the runtime!" unless $runtime;
```

Comment:
Good work Pavel!

Probably here `//= "docker"` has to be removed, because of the condition below?

Reply:
Thank you Anna 😄